sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -148,14 +148,41 @@ def configure_span_processors
       end
 
       def default_span_processor
-        Trace::Export::SimpleSpanProcessor.new(
+        Trace::Export::SimpleSpanProcessor.new(configured_exporter)
+      end
+
+      def configured_exporter
+        exporter = ENV.fetch('OTEL_TRACES_EXPORTER', 'otlp')
+        case exporter
+        when 'otlp' then otlp_exporter
+        when 'jaeger' then jaeger_exporter
+        when 'zipkin' then zipkin_exporter
+        else
+          OpenTelemetry.logger.warn "The #{exporter} exporter is unknown and cannot be configured, configuring the console exporter instead"
           Trace::Export::ConsoleSpanExporter.new
```

Comment:
I'm wondering if we should not just fail to initialize here, this has some implication that a value was set, but is incorrect.  

I'm considering the situation where an application that relies on the env to configure their tracing exporter has a typo in the env and suddenly all of their logs are flooded with traces.

Reply:
I've opted not to "fail", but instead log and fall back to the NoopSpanProcessor